src/components/Navbar.tsx

```diff
@@ -5,30 +5,25 @@ import { useState } from "react";
 import Link from "next/link";
 import { Equal, X } from "lucide-react";
 import { Button } from "@/components/ui/button";
+import PromochatorLogo from "../../public/assets/logo/promochator-logo.svg"
 
 export function Navbar() {
   const [showSidebar, setShowSidebar] = useState<boolean>(false);
   return (
     <>
-      <div className="w-full flex flex-row justify-between items-center gap-12">
-        <div className="flex flex-row gap-2 items-center mr-auto">
-          <Image
-            src="/assets/logo/promochator_logo.png"
-            alt="Promochator Logo"
-            className="w-6"
-            width={150}
-            height={150}
-          />
-          <p className="text-lg font-semibold">
-            Promo<span className="text-[#5f7ecd]">CHAT</span>or
-          </p>
-        </div>
+      <div className="flex justify-between items-center">
+        <Image
+          src={PromochatorLogo}
+          alt="logo Promochatora"
+          width={128}
```

Comment:
PROMOCHATOR logo is too small. Ideally, the size would be equal to the KN SOLVRO logo.

Reply:
I agree. Changed it from `128x64` to `160x34` in https://github.com/Solvro/web-promochator/pull/17/commits/8a8a4c2b6ce00645b71e26c4ab894545d6422d61. I think it looks better now